templates/email_templates.py

```diff
@@ -0,0 +1,17 @@
+sender_email = 'woodworkssatish@gmail.com'
+sender_key = "oonweszigkvauwdm"
+
+verification_email_subject = "Your Tehelka Verification Code"
+
+verification_email_body = """
+Hello and welcome to Tehelka App!
+
+Thank you very much for signing up. To ensure the security of your account, we have generated a verification code for you: XXXXXX. Please use this code to verify your email address using our website.
```

Comment:
Instead of "XXXXXX", use `{verification_code}` and then this string can be formatted. See other comment.